libevm/VMOpt.cpp

```diff
@@ -49,20 +49,22 @@ void VM::initMetrics()
 	done = true;
 }
 
-//
-// init interpreter on entry
-//
-
+/// Init interpreter on entry.
 void VM::initEntry()
 {
-	m_bounce = &VM::interpretCases;	
+	m_bounce = &VM::interpretCases;
+
+	// Copy and extend code by 32 bytes to handle virtual push data.
```

Comment:
@chriseth the copy happens in the assignment on line 60.  My line comments above explain what is happening, I think, so could be pulled into the comment.  And of course Pawel knows what he was doing.  It should be 33 bytes, not 32, as @chfast said yesterday.